<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMianWind</class>
 <widget class="QMainWindow" name="QWMianWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMianWind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actFileOpen_2"/>
    <addaction name="actFileOpen"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actCut"/>
    <addaction name="actPaste"/>
    <addaction name="actCopy"/>
    <addaction name="separator"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actBold"/>
    <addaction name="actItalic"/>
    <addaction name="actUnderline"/>
    <addaction name="separator"/>
    <addaction name="actToolbarLab"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFileOpen_2"/>
   <addaction name="actFileOpen"/>
   <addaction name="actExit"/>
   <addaction name="actCopy"/>
   <addaction name="actCut"/>
   <addaction name="actPaste"/>
   <addaction name="actClear"/>
   <addaction name="actBold"/>
   <addaction name="actItalic"/>
   <addaction name="actUnderline"/>
   <addaction name="action"/>
  </widget>
  <action name="actCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/cut.png</normaloff>:/images/Icon/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/copy.png</normaloff>:/images/Icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/paste.png</normaloff>:/images/Icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/bold.png</normaloff>:/images/Icon/bold.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/italic.png</normaloff>:/images/Icon/italic.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/underline.png</normaloff>:/images/Icon/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/exit.png</normaloff>:/images/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actFileOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/openfile.png</normaloff>:/images/Icon/openfile.png</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFileOpen_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/newfile.png</normaloff>:/images/Icon/newfile.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/clear.png</normaloff>:/images/Icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/fontset.png</normaloff>:/images/Icon/fontset.png</iconset>
   </property>
   <property name="text">
    <string>字体...</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actToolbarLab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>QWMianWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
